scenario/utils/index.ts

```diff
@@ -352,12 +352,13 @@ export async function executeOpenProposal(
     await governor.queue(id, { gasPrice: 0 });
   }
 
-  const proposal = await governor.proposals(id);
-  await setNextBlockTimestamp(dm, proposal.eta.toNumber() + 1);
-
-  // Execute proposal (w/ gas limit so we see if exec reverts, not a gas estimation error)
-  await setNextBaseFeeToZero(dm);
-  await governor.execute(id, { gasPrice: 0, gasLimit: 12000000 });
+  // Execute proposal (maybe, w/ gas limit so we see if exec reverts, not a gas estimation error)
+  if (state == ProposalState.Queued) {
```

Comment:
This is probably the breaking change. You'll want to set `state` as `Queued` after queuing it in the block above if we add this check.